### PR TITLE
[Releng] Support rocky8 platform on gpdb7

### PR DESCRIPTION
**Pull request by BaiShaoqi**
## Here are some reminders before you submit the pull request
- [ ] Add tests for the change
- [ ] Document changes
- [ ] Communicate in the mailing list if needed
- [ ] Pass `make installcheck`
- [ ] Review a PR in return to support the community

### CODE REVIEW

File: concourse/scripts/common.bash [hunk 1]
```diff
@@ -46,7 +46,7 @@ function configure() {
     # The full set of configure options which were used for building the
     # tree must be used here as well since the toplevel Makefile depends
     # on these options for deciding what to test. Since we don't ship
-    ./configure --prefix=/usr/local/greenplum-db-devel --disable-orca --enable-gpcloud --enable-mapreduce --enable-orafce --enable-tap-tests --with-gssapi --with-libxml --with-openssl --with-perl --with-python --with-uuid=e2fs PYTHON=python3 PKG_CONFIG_PATH="${GPHOME}/lib/pkgconfig" ${CONFIGURE_FLAGS}
+    ./configure --prefix=/usr/local/greenplum-db-devel --disable-orca --enable-gpcloud --enable-mapreduce --enable-orafce --enable-tap-tests --with-gssapi --with-libxml --with-openssl --with-perl --with-python PYTHON=python3 PKG_CONFIG_PATH="${GPHOME}/lib/pkgconfig" ${CONFIGURE_FLAGS}
 
     popd
```
Comment (50wu) [inline concourse/scripts/common.bash:51]: It should confirm with extension team if we remove the `--with-uuid=e2fs`

Comment (beeender) [reply concourse/scripts/common.bash:51]: `--with-uuid=e2fs` should not be removed. We decided to support `contrib/uuid-ossp` officially. `--with-uuid=e2fs` is the enabler of the extension.

Comment (BaiShaoqi) [reply concourse/scripts/common.bash:51]: Thanks @50wu !

**Review by lucasbonner — APPROVED** [review]
Looks great!

**Review by khuddlefish — APPROVED** [review]

**Pull request merged**